src/onboarding/Deployment/DeploymentStepsPanel.js

```diff
@@ -65,10 +68,19 @@ function DeploymentStepsPanel({ transactionsStatus, pending, allSuccess }) {
       </div>
 
       {!allSuccess && (
-        <Info>
+        <Info mode="warning">
           It might take some time before these transactions get processed.
-          Please be patient and{' '}
-          <strong>do not close this window until it finishes.</strong>
+          Please be patient and do not close this window until it finishes.
+          {provider === 'metamask'
+            ? ' Additionally, do not use the "Speed Up" MetaMask feature, otherwise it will result in configuration errors. '
+            : ' '}
+          For more details,{' '}
+          <Link
+            href="https://help.aragon.org/category/24-troubleshooting"
```

Comment:
My only issue with this link is that we don't have an article in this help section that describes this case.

Can you submit a PR to https://github.com/aragon/help with the relevant article and then link this to that article?

Reply:
@sohkai We'll be submitting a PR soon, here's the [draft text](https://hackmd.io/jNBJEAfDRhealFnxe1k7_Q) in progress.

Reply:
@sohkai here's the PR: https://github.com/aragon/help/pull/13

Is there a way to already know this page's URL? Maybe something like https://help.aragon.org/article/39-launch-taking-a-long-time-to-process ?